modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -1799,7 +1807,7 @@ void ONNXImporter::parseLSTM(LayerParams& layerParams, const opencv_onnx::NodePr
 
     bool need_yc = lstm_proto.output_size() > 2 && !lstm_proto.output(2).empty();
     bool need_yh = lstm_proto.output_size() > 1 && !lstm_proto.output(1).empty();
-    bool need_y = lstm_proto.output_size() > 0 && !lstm_proto.output(0).empty();
+    bool need_y  = lstm_proto.output_size() > 0 && !lstm_proto.output(0).empty();
```

Comment:
please revert

Reply:
Done! But why?

Reply:
Just simpler for the future tracking of changed lines.